src/molecules/formfields/GoogleAutoCompleteField/__tests__/index.spec.js

```diff
@@ -106,6 +106,53 @@ describe('<GoogleAutoCompleteField />', () => {
       expect(props.autocompleteAddressFields.mock.calls.length).toBe(1);
       expect(props.autocompleteAddressFields.mock.calls[0][0]).toEqual(expectedData);
     });
+
+    describe('when google does not provide street_number', () => {
+      let addressValuesWithoutStreetNumber;
+      let state;
+
+      beforeEach(() => {
+        addressValuesWithoutStreetNumber = addressValues
+          .filter(value => !value.types.includes('street_number'));
+
+        state = {
+          autocomplete: {
+            getPlace: jest.fn(() => ({
+              address_components: addressValuesWithoutStreetNumber,
+            }))
+          }
+        };
+      });
+
+      it('should extract the street_number manually', () => {
+        const streetNumber = 1234;
+        const id = 'autocomplete-id';
+        const input = { name: 'address', value: `${streetNumber} Broadw` };
+        const wrapper = shallow(<GoogleAutoCompleteField {...props} {...{ id, input }} />);
```

Comment:
Just a suggestion, but another way that I've updated the props is something like: 
```
const testProps = {
   ...props,
   id,
   input,
};
<Component {...testProps} />
```

Reply:
Yeah that looks a little more standard, I'll do that!